htlcswitch/link.go

```diff
@@ -1104,10 +1108,17 @@ func (l *channelLink) htlcManager() {
 				continue
 			}
 
+			// For normal channels, we use a conf target of 3 to
+			// determine the current network fee.
+			confTarget := uint32(3)
```

Comment:
nit: would be cool to have also a constant for this variable like for example: defaultCommitmentConfTarget = 3? Why are we using 3, is it just a common agreement ?

Reply:
good idea, I will move it out to a constant. 
Yes i think using 3 was just the norm.